src/tournament-mode/RoundSelect.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import { Button, MenuItem } from "@blueprintjs/core";
+import { Select2 } from "@blueprintjs/select";
+
+import { filterRoundLabel, renderRoundLabel, IRoundLabel, roundLabels } from "./round-label";
+
+const RoundLabel = Select2.ofType<IRoundLabel>();
+
+export const RoundSelect: React.FC = () => {
+  const [roundLabel, setRoundLabel] = useState<IRoundLabel>(roundLabels[0]);
```

Comment:
Rather than save this as state local to this component, we need to save this to the state of the drawing (which will then cause it to sync over the network, when enabled)